index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="HTML, CSS, Javascript">
+    <meta name="author" content="Maen Kh">
+    <title>Global Warmming</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="components/header.js"  defer></script>
+    <script src="components/footer.js"  defer></script>
+    <script src="/JS/javascript.js"></script>
+    
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+    <header-component></header-component>
+    <main class="main-section">
+        <a class="btn-hamburger" href="#">
+            <div class="main-header-hamburger">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </a>
+        <ul class="hamburger-menu">
```

Comment:
- [ ] When I open the mobile menu and switch to the desktop screen the mobile menu is still on the screen it should disappear. Please make sure when we switch the screen from mobile to desktop the mobile menu should disappear.
 - You can use media queries for that if the screen cross the min-width you can set your mobile menu `display: none`

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/158020287-5bc79c03-b424-4910-ad18-b100b857fd67.png)